### PR TITLE
[CS] Code Style fixes for administrator/components/com_installer/

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for Issue code style fixes

### Summary of Changes
- Multi-line function call not indented correctly;
- No blank line found after control structure
- Closing parenthesis of a multi-line function call must be on a line by itself
- Please consider an empty line before the foreach statement;
- Expected 1 space after "="; 2 found
- Expected 1 space after FUNCTION keyword; 0 found
- Expected 3 tabs before opening brace; 4 found
- Whitespace found at end of line

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to implement fixable code style

### Documentation Changes Required
none

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 629aa0fe98acb812e0fd2a9898e294d50b6e7c5e<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16916">issues.joomla.org/tracker/joomla-cms/16916</a>.</sub>

**Comment by photodude** [comment]
@mbabker @wilsonge bump for final review and merge consideration

File: administrator/components/com_installer/models/database.php [hunk 1]
```diff
@@ -310,15 +311,18 @@ private function prepareUtf8mb4StatusTable()
 		{
 			// Table messed up somehow, clear it
 			$db->setQuery('DELETE FROM ' . $db->quoteName('#__utf8_conversion')
-				. ';')->execute();
+				. ';'
+			)->execute();
```
Comment (wilsonge) [inline administrator/components/com_installer/models/database.php]: 😢  This looks awful visually. Can the semicolon not go on the line above?

Comment (photodude) [reply administrator/components/com_installer/models/database.php]: My personal preference would be something like
```php
$db
	->setQuery('DELETE FROM ' . $db->quoteName('#__utf8_conversion') . ';')
	->execute();
```
This is a good example showing either a problem with the autofixer or something that should be adjusted manually after the autofixers have run. Like all automated tools, nothing is perfect. 

I've avoided "fixing things" manually as this PR is focused on just testing the autofixers. If necessary for merge consideration I'll adjust this. but I would rather that its merged only as the autofixer presented it.

Comment (wilsonge) [reply administrator/components/com_installer/models/database.php]: OK I've manually fixed that one. The code sniffer correctly identified an issue. I don't mind us manually tweaking things to make it work

**Comment by photodude** [comment]
Thanks @wilsonge

**Pull request merged**